### PR TITLE
Css flexbox. -Euichang Park

### PR DESCRIPTION
**Pull request by EuichangPark**
alignitems
flexwrapper
justifycontent

### CODE REVIEW

File: css-flexbox/align-items.html [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <meta http-equiv="X-UA-Compatible" content="ie=edge">
+  <title>CSS Flexbox - align-items</title>
+  <link rel="stylesheet" href="reset.css">
+  <link rel="stylesheet" href="pokemon.css">
+  <style>
+    /* your styles here 👇 */
+    #card-deck {
+      display: flex;
+      flex-basis: calc(100% / 3);
```
Comment (Zhiwutian) [inline css-flexbox/align-items.html:14]: Flex basis is a flex item property, so it needs to be used on the `card-wrapper` class

**Review by Zhiwutian — CHANGES_REQUESTED** [review]
a couple small updates!

**Comment by EuichangPark** [comment]
alignitems2
justfycontent2

**Comment by EuichangPark** [comment]
alignitems3

**Review by Zhiwutian — APPROVED** [review]
👍🏼

**Pull request merged**